apps/snitch_core/lib/core/domain/order/default_machine.ex

```diff
@@ -57,16 +57,17 @@ defmodule Snitch.Domain.Order.DefaultMachine do
 
   state_machine OrderSchema,
                 :state,
-                ~w(cart address payment processing rts shipping complete cancelled)a do
+                ~w(cart address payment delivery processing rts shipping complete cancelled)a do
     event(:add_addresses, %{from: [:cart], to: :address}, fn context ->
       context
       |> Transitions.associate_address()
       |> Transitions.compute_shipments()
       |> Transitions.persist_shipment()
     end)
 
-    event(:add_payment, %{from: [:address], to: :payment}, fn context ->
+    event(:save_packages_methods, %{from: [:address], to: :delivery}, fn context ->
```

Comment:
IMO `add_shipping_preferences` was fine, `save_shipping_preferences` is also fine.

Reply:
yes, the naming should be around shipping methods. `shipping` word in the name is really important here.

Reply:
`save_shipping_preferences` is appropriate